psi-ms.obo

```diff
@@ -22682,6 +22682,35 @@ relationship: has_optional_column: MS:1000767 ! native spectrum identifier forma
 relationship: has_column: UO:0000191 ! fraction
 relationship: has_relation MS:1000285 ! total ion current
 
+[Term]
+id: MS:4000069
+name: m/z acquisition range
+def: "Upper and lower limit of m/z precursor values at which MSn spectra are recorded." [PSI:QC]
+is_a: MS:4000004 ! n-tuple
+relationship: has_metric_category MS:4000009 ! ID free metric
+relationship: has_metric_category MS:4000012 ! single run based metric
+relationship: has_units MS:1000040 ! m/z
+
+[Term]
+id: MS:4000070
+name: retention time acquisition range
+def: "Upper and lower limit of time at which spectra are recorded." [PSI:QC]
```

Comment:
I'd again explicitly specify "retention time" here to clearly disambiguate from wall time (or other times).